ppr-ui/src/components/mhrRegistration/ReviewConfirm/SubmittingPartyReview.vue

```diff
@@ -16,34 +16,139 @@
+                </v-col>
+              </v-row>
+            </v-col>
+            <v-col cols="3" class="pl-1">
+              <base-address
+                v-if="hasAddress"
+                class="content"
+                :schema="addressSchema"
+                :value="address"
+              >
+              </base-address>
+              <p v-else class="content"> (Not Entered) </p>
+            </v-col>
+            <v-col cols="3" class="pl-3">
+              <p class="content">{{getMhrRegistrationSubmittingParty.emailAddress || '(Not Entered)'}}</p>
+            </v-col>
+            <v-col cols="3" class="pl-4">
+              <p class="content" v-html="parsePhoneNumber()"></p>
+            </v-col>
+          </v-row>
+
+          <div class="px-4">
+            <v-divider />
+          </div>
+
+          <v-row no-gutters class="px-6 py-7">
+            <v-col cols="3">
+              <p class="side-header">Attention or<br>Reference Number</p>
+            </v-col>
+            <v-col cols="9">
+              <p class="content">{{getMhrAttentionReferenceNum || '(Not Entered)'}}</p>
+            </v-col>
+          </v-row>
         </section>
       </template>
     </div>
   </v-card>
 </template>
 
 <script lang="ts">
-import { defineComponent, reactive, toRefs } from '@vue/composition-api'
+import { computed, defineComponent, reactive, toRefs } from '@vue/composition-api'
 import { RouteNames } from '@/enums'
+import { useGetters } from 'vuex-composition-helpers'
+import { BaseAddress } from '@/composables/address'
+import { PartyAddressSchema } from '@/schemas'
 
 export default defineComponent({
   name: 'SubmittingPartyReview',
-  components: {},
+  components: {
+    BaseAddress
+  },
   props: {},
   setup () {
-    const localState = reactive({})
+    const {
+      getMhrRegistrationSubmittingParty,
+      getMhrAttentionReferenceNum
+    } = useGetters<any>([
+      'getMhrRegistrationSubmittingParty',
+      'getMhrAttentionReferenceNum'
+    ])
+
+    const localState = reactive({
+      address: computed(() => getMhrRegistrationSubmittingParty.value.address),
+      businessName: computed(() => getMhrRegistrationSubmittingParty.value.businessName),
+      personName: computed(() => getMhrRegistrationSubmittingParty.value.personName),
+      hasAddress: computed(() => !Object.values(localState.address).every(val => !val))
+    })
+
+    const addressSchema = PartyAddressSchema
+
+    const parsePhoneNumber = () => {
+      const phone = getMhrRegistrationSubmittingParty.value
+      const phoneNum = phone.phoneNumber
+      const ext = phone.phoneExtension ? ' &nbsp;Ext ' + phone.phoneExtension : ''
+      return phoneNum ? `${phoneNum}${ext}` : '(Not Entered)'
+    }
+
+    const getSubmittingPartyName = () => {
+      if (!localState.businessName && Object.values(localState.personName).every(val => !val)) {
+        return '(Not Entered)'
+      } else {
+        if (localState.businessName) {
```

Comment:
I think this will always be true since our model is initially set as businessName: ''. What you think @cameron-freshworks . cc: @garyfan1

Reply:
I believe an empty string in typescript resolves as `falsey` 
So `businessName:  ''` being initialized would still equate too `!localState.businessName` ie `false` 
@garyfan1 Should/could verify though

Reply:
I was thinking empty string is falsy? so even if `business == ''`, `if(business)` is still false?

Reply:
@cameron-freshworks right, you are correct. Resolving this.